distributed/client.py

```diff
@@ -343,7 +343,7 @@ class Client(object):
     distributed.scheduler.Scheduler: Internal scheduler
     """
     def __init__(self, address=None, start=True, loop=None, timeout=3,
-                 set_as_default=True, scheduler_file=None):
+                 set_as_default=True, scheduler_file=None, connection_kwargs=None):
```

Comment:
Instead of passing `connection_kwargs` everywhere (here, in the scheduler, in the workers...), perhaps those parameters should be stored as configuration options? That's what we do for other global config options.